lib/super_diff/colorized_document_extensions.rb

```diff
@@ -7,11 +7,11 @@ def self.extended(extendee)
     end
 
     def alpha(*args, **opts, &block)
-      colorize(*args, **opts, fg: SuperDiff::COLORS.fetch(:alpha), &block)
+      colorize(*args, **opts, fg: SuperDiff.configuration.alpha_color, &block)
```

Comment:
If these are configurable now, I think it makes more sense to stick with one name rather than have a public name and a private name. So I would vote for this becoming `expected_color`. (I can take care of replacing all instances of "alpha" with "expected" in tests and such.)

Same for `beta` below.

Reply:
I am fine with that if you want to do that.